attributes/default.rb

```diff
@@ -238,26 +245,55 @@
   'default' => "659400f9842fffb8d64ae0b650f081b9",
   'ubuntu' => { 'default' => "c2b410031867dc7c966ca5b1aa0c72aa" }
 )
+
+# MySQL Packages
```

Comment:
@davprat please let's put a comment saying that we install our pinned version of the MySQL packages in every combination of OS/architecture except for Ubuntu ARM (where we rely in the OS repositories).